src/guides/v2.4/install-gde/prereq/php-settings.md

```diff
@@ -41,18 +41,25 @@ Magento requires a set of extensions to be installed:
 <!--{% assign packages = site.data.codebase.v2_4.open-source.composer_lock.packages %}-->
 {% include install/php-extensions-template.md %}
 
-In the command line, type:
+{:.bs-callout-warning}
+If you install Magento by cloning the [magento/magento2](https://github.com/magento/magento2) GitHub repository, you must have the [ext-sockets](https://github.com/php-amqplib/php-amqplib/blob/master/CHANGELOG.md#281---2018-11-13) extension installed on your system.
```

Comment:
This is incorrect information for Magento 2.4.0. It's correct for Magento 2.3.x, but not for 2.4.x. The `sockets` plugin is **always** needed for all installation scenario's, not only when you clone Magento's repository.

Reply:
This seems to reverts https://github.com/magento/devdocs/pull/7398, which is not correct.